src/strategies/polygon/sushiswap/strategy-sushi-usdc-bct.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-sushi-farm-base.sol";
+
+contract StrategySushiUsdcBctLp is StrategySushiFarmBase {
+    // Usdc/Bct pool id in MasterChef contract
+    uint256 public sushi_usdc_bct_poolId = 37;
+
+    //Token addresses
+    address public usdc_bct_lp = 0x1E67124681b402064CD0ABE8ed1B5c79D2e02f64;
+    address public usdc = 0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174;
+    address public bct = 0x2F800Db0fdb5223b3C3f354886d907A671414A7F;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategySushiFarmBase(
+            usdc,
+            bct,
+            sushi_usdc_bct_poolId,
+            usdc_bct_slp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    // ***** Views *****
+
+    function getName() external pure override returns (string memory) {
+        return "StrategySushiUsdcBctLp";
+    }
+
+    // ***** State Mutations *****
+
+    function harvest() public override onlyBenevolent {
+        // Collects SUSHI tokens
+        IMiniChefV2(miniChef).harvest(poolId, address(this));
+        uint256 _sushi = IERC20(sushi).balanceOf(address(this));
+        if (_sushi > 0) {
+            _swapSushiswap(sushi, weth, _sushi);
+        }
+
+        // Collect MATIC tokens
```

Comment:
We're only concerned with BCT tokens here. WMATIC doesn't come into play. Path will be BCT -> USDC -> WETH